frontend/app/MediaPage.tsx

```diff
@@ -64,11 +64,11 @@ export default function MediaPage() {
             paddingBottom: height - insets.top, // Add padding at the bottom equal to the height of the bottom tab bar
           }}
         >
-          <ThemedView>
+          <View>
```

Comment:
change ThemedView to View so that the color is consistent whether light mode or dark mode